src/components/Auth.elements.tsx

```diff
@@ -94,8 +107,36 @@ export const SignUpBtn = styled.button`
   border: none;
   cursor: pointer;
 `;
+export const AuthSelect = styled.select`
```

Comment:
did you see any styling issue with just doing styled components like this?

Reply:
The 'styled.select' component doesn't allow for the drop-down field to be styled. I am not sure how to proceed with setting up React-Select as it requires integration to the API to get instructor data for the drop-down options. Other than this, I didn't notice anything that seemed out of place.

Reply:
should revisit this after we successfully connect to the backend? It would be nice to see the drop-down contents to realize what we're working with